dojo.py

```diff
@@ -1,12 +1,103 @@
+from random import choice
+from rooms import Room, Office, LivingSpace
+from person import Person, Fellow, Staff
+
+
 class Dojo():
 
 	def __init__(self):
-		pass
+		self.all_rooms = []
+		self.offices = []
+		self.livingSpaces = []
+		self.all_people = []
+		self.fellows = []
+		self.staffs = []
 
-	def create_room(self, room_name, room_type):
+	def create_room(self, room_type, room_names):
 		""" function to create a unique room space """
-		pass
-		
-	def add_person(self, name, category, accomodation="N"):
+		msg = ''
+		for name in room_names:
+			room_name = name.lower()
+			room_type = room_type.lower()
+
+			found = False
+			for room in self.all_rooms:
+				if room_name == room.room_name:
+					found = True
+			if found:
+				msg += "Room with name " + room_name + " already exists \n"
+			else:
+				if room_type in ('office', 'living'):
+					room = Room(room_name)
+					self.all_rooms.append(room)
+				if room_type == 'office':
+					room = Office(room_name)
+					self.offices.append(room)
+					msg += "Office " + room_name + " added successfully \n"
+				elif room_type == 'living':
+					room = LivingSpace(room_name)
+					self.livingSpaces.append(room)
+					msg += "Living space " + room_name + " added successfully\n"
+				else:
+					msg += "Wrong room type \n"
+			return msg
+
+
+	def add_person(self, first_name, last_name, category, accomodation="N"):
 		""" function to add a person """
-		pass
+		full_name = first_name.capitalize() + " " + last_name.capitalize()
+		category = category.lower()
+		accomodation = accomodation.upper()
+
+		if category in ("fellow", "staff"):
+
+			id = len(self.all_people) + 1
+			person = Person(first_name, last_name)
```

Comment:
from your person model, Person class accepts two values, name and category. Here you're passing the first name and last name to an instance of this class